### ISSUE TITLE
Order Correction Changes

### ISSUE THREAD

**Issue by Rajesh8111** [Rajesh8111/FAS#13]
**First Mail from Client**

From: Sheets, Beverly [mailto:Beverly.Sheets@windstream.com] 
Sent: 12 July 2018 00:00
To: Arunkumar V K
Cc: Goble, Eric; Fairchild, Kathy
Subject: Order Corrections Scenarios & Updates

Arun,

We have made some updates to the Order Corrections Quick Reference Guide over the past few weeks.  All Scenarios were updated to remove all of the symbols that cannot be used in the OC Tool.  The below scenarios were added or updated.  Everything highlighted in Blue was added to the process.  

Order   Corrections   ‘Update’ | Procedures/Assignment/Order   Corrections    Procedures/Service   Activation/Order Corrections                     Description               Any DCRIS service order or Remedy     Request that cannot be assigned by the due date, as a result of incorrect     information or lack of information, will be referred to the appropriate     group. Orders may be referred for address correction, apartment #, lot #, 911     Address, ASOC errors and various Order Corrections.            The Assignment Department and Service     Activation Group (SAG) will submit all order corrections to the Order     Corrections Group using the Order Corrections Form.           The     following symbols cannot be used entered in the OC Tool:  “ “ ;  ‘ ~ `     ^ { } [ ] \ \| <  >                     NOTE: The   Assignment and Service Activation Quick Reference Guides Comments section   have been to updated to remove these special chatacters so the copy pasted   function can be used. Reminder, an test in red needs replaced in the OC tool   with the info pertaining to that order. | Description | Any DCRIS service order or Remedy     Request that cannot be assigned by the due date, as a result of incorrect     information or lack of information, will be referred to the appropriate     group. Orders may be referred for address correction, apartment #, lot #, 911     Address, ASOC errors and various Order Corrections.            The Assignment Department and Service     Activation Group (SAG) will submit all order corrections to the Order     Corrections Group using the Order Corrections Form.           The     following symbols cannot be used entered in the OC Tool:  “ “ ;  ‘ ~ `     ^ { } [ ] \ \| <  >
-- | -- | -- | --
Description | Any DCRIS service order or Remedy     Request that cannot be assigned by the due date, as a result of incorrect     information or lack of information, will be referred to the appropriate     group. Orders may be referred for address correction, apartment #, lot #, 911     Address, ASOC errors and various Order Corrections.            The Assignment Department and Service     Activation Group (SAG) will submit all order corrections to the Order     Corrections Group using the Order Corrections Form.           The     following symbols cannot be used entered in the OC Tool:  “ “ ;  ‘ ~ `     ^ { } [ ] \ \| <  >

**Comment by Rajesh8111** [comment]
As per rajkiran says, 

This process change will be confirmed only after 18-07-18.

Kept on Hold..

**Comment by Rajesh8111** [comment]
Sl # | CHRONOS Options | Input field
-- | -- | --
1 | Order Entry-SSRV(No Maintenance   line) | Yes
2 | Order Entry-SSRV - BB speed   required VoIP |  
3 | ASOC - [SSRV] - Consumer VoIP   Access Line |

**Comment by Rajesh8111** [comment]
[Order Correction for 60 & 80 status.xlsx](https://github.com/Rajesh8111/FAS/files/2210294/Order.Correction.for.60.80.status.xlsx)
